src/org/zaproxy/zap/view/HttpPanelManager.java

```diff
@@ -112,7 +112,12 @@ public void removeRequestViews(String componentName, String viewName, Object opt
 	public void removeRequestDefaultViewSelectorFactory(String componentName, String defaultViewSelectorFactoryName) {
 		requestPanels.removeDefaultViewSelectorFactory(componentName, defaultViewSelectorFactoryName);
 	}
-	
+
+	public void removeRequestDefaultViewSelectors(String componentName, String defaultViewSelectorName, Object options) {
+		requestPanels.removeDefaultViewSelectors(componentName, defaultViewSelectorName, options);
+	}
+
+	@Deprecated
```

Comment:
Please also add a `@deprecated` JavaDoc tag pointing to new method (also add `(TODO add version)` at the beginning to later indicate the version in which was deprecated).